miles/test_draw_hov_with_time_axis.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
+import sys, os
+sys.path.insert(1,'../')
+from utils.vvmtools import newVVMtools
+from utils.plottools import dataPlotters
+import config
+import matplotlib.pyplot as plt
+
+import numpy as np
+import sys, os
+sys.path.insert(1,'../')
+from utils.vvmtools import newVVMtools
+import config
+
+if __name__=='__main__':
+    exp = 'op_1'
+    path = f'{config.vvmPath}/{exp}/'
+    figpath = f'./fig/'
+    vvm = newVVMtools(path)
+    # test example for
+    reg       = 'all' #'left', 'right'
+    drange = vvm.get_domain_range(reg)
+    '''
+    ens = vvm.func_time_parallel(\
+          func       = vvm.cal_enstrophy,\
+               time_steps = np.arange(vvm.nt), \
+               func_config = {'domain_range':drange,},\
+               cores = 5,\
+              ).T
+    print(ens.shape)
+
+    pblh_ens = vvm.func_time_parallel(\
+               func       = vvm.cal_pblh_ens, \
+               time_steps = np.arange(vvm.nt), \
+               func_config = {'domain_range':drange, 'threshold':1.e-5},\
+               cores = 5,\
+              )
+    print(pblh_ens.shape)
+    pblh_tke = vvm.func_time_parallel(\
+               func       = vvm.cal_pblh_tke, \
+               time_steps = np.arange(vvm.nt), \
+               func_config = {'domain_range':drange, 'threshold':0.1},\
+               cores = 5,\
+              )
+    pblh_th0p5 = vvm.func_time_parallel(\
+                 func       = vvm.cal_pblh_0p5, \
+                 time_steps = np.arange(vvm.nt), \
+                 func_config = {'domain_range':drange,},\
+                 cores = 5,\
+              )
+    pblh_maxgrad = vvm.func_time_parallel(\
+                 func       = vvm.cal_pblh_maxgrad, \
+                 time_steps = np.arange(vvm.nt), \
+                 func_config = {'domain_range':drange,},\
+                 cores = 5,\
+              )
+    '''
+    tr01=np.load('./data/tr01.npy')
+
+    data_dims = {'x':vvm.DIM['xc']/1e3,\
+                 'y':vvm.DIM['yc']/1e3,\
+                 'z':vvm.DIM['zc']/1e3,\
+                 't':[np.datetime64('2024-01-01 05:00:00')\
```

Comment:
This time dim can add into utils/vvmtools in the future.
and i prefer this method to create time array
`np.datetime64('2024-01-01 05:00:00') + np.arange(total_time) * np.timedelta64(delta_min,'m')` ~

Reply:
Agree!